DOCUMENTATION.md

```diff
@@ -487,6 +488,10 @@ A unique identifier you create. This reference identifies a single unique notifi
 "your_reference_here"
 ```
 
+#### postage (optional)
```

Comment:
Similar to PHP client, I think these should be ordered the same way as the function arguments